dive_core/dive_file_container_private.h

```diff
@@ -0,0 +1,44 @@
+/*
+ Copyright 2025 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+namespace Dive
+{
+
+struct DiveFileContainerHeader
+{
+    // Unix Standard Tar compatible format:
```

Comment:
better to add a ref link so that people know where to find the format
(i guess here? https://www.ibm.com/docs/en/zos/2.4.0?topic=formats-tar-format-tar-archives)

Reply:
The actual reference is POSIX IEEE P1003.1, which is behind subscription, otherwise Wikipedia's [Tar](https://en.wikipedia.org/wiki/Tar_(computing)) gives reasonable explaination.